catalog/datasets/CM2.6 4xCO2.yaml

```diff
@@ -0,0 +1,23 @@
+name: "CM2_6 4xCO2"
+description: "GFDL CM2.6 4xCO2 Ocean Simulation"
+stores:
+  - name: "GFDL_CM2_6_CONTROL_5-DAY_FIELD2023-10-15 12:24:24.049838.zarr"
+    href: "leap-persistent-ro/groundpepper/GFDL_cm2.6_4xCO2/GFDL_CM2_6_CONTROL_5-DAY_FIELD2023-10-15 12:24:24.049838.zarr"
+  - name: "GFDL_CM2_6_CONTROL_DAILY_100M2023-10-15 20:55:40.830261.zarr"
+    href: "leap-persistent-ro/groundpepper/GFDL_cm2.6_4xCO2/GFDL_CM2_6_CONTROL_DAILY_100M2023-10-15 20:55:40.830261.zarr"
+  - name: "GFDL_CM2_6_CONTROL_DAILY_SURF2023-10-16 14:51:44.387783.zarr"
+    href: "leap-persistent-ro/groundpepper/GFDL_cm2.6_4xCO2/GFDL_CM2_6_CONTROL_DAILY_SURF2023-10-16 14:51:44.387783.zarr"
+  - name: "GFDL_CM2_6_CONTROL_MONTHLY_ADVTERM2023-10-15 18:12:37.539380.zarr"
+    href: "leap-persistent-ro/groundpepper/GFDL_cm2.6_4xCO2/GFDL_CM2_6_CONTROL_MONTHLY_ADVTERM2023-10-15 18:12:37.539380.zarr"
+  - name: "GFDL_CM2_6_CONTROL_MONTHLY_UPPER100M2023-10-15 13:23:16.373001.zarr"
+    href: "leap-persistent-ro/groundpepper/GFDL_cm2.6_4xCO2/GFDL_CM2_6_CONTROL_MONTHLY_UPPER100M2023-10-15 13:23:16.373001.zarr"
```

Comment:
@andersy005 is there a way to display entries in a nested/collapsible form here? 

Basically I would like for all stores here to be in a single catalog entry like this:

- CM2-0:
    - CM2.6:
       - Control
           - Surface
           - 3D
           - ...
       - 2xCO2
           - Surface 
           - 3D
           - ...
       - ...
    - ...

Reply:
>  is there a way to display entries in a nested/collapsible form here?

unfortunately, due to the way the schema is configured, this needs to a flat list of dicts.

> Basically I would like for all stores here to be in a single catalog entry like this:

all entries within the same YAML file will be in a single catalog entry. can you expand on what you mean by this ☝🏽? perhaps i'm misunderstanding the question. 

e.g. https://github.com/leap-stc/data-management/blob/main/catalog/datasets/large_ensemble_pco2_testbed.yaml

![Screenshot 2023-11-28 at 13 59 55](https://github.com/leap-stc/data-management/assets/13301940/4a289b9e-ad78-416e-82cc-352f613ce1d1)